packages/render-fidelity-tools/test/config.json

```diff
@@ -25,6 +25,20 @@
     }
   ],
   "scenarios": [
+    {
+      "name": "khronos-DragonAttenuation",
+      "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/DragonAttenuation/glTF/DragonAttenuation.gltf",
+      "dimensions": {
+        "height": 120,
+        "width": 130
```

Comment:
Sorry, this wasn't what I meant. Let's keep the resolution around what we usually use; I just don't want lots of blank pixels. For this one, I'm realizing it'll probably make more sense to zoom in a bit as we don't need to see the whole drape. Let's just focus on the dragon and frame that.